xml/admin_monitoring.xml

```diff
@@ -96,5 +95,435 @@ https://raw.githubusercontent.com/kubernetes/heapster/release-1.3/deploy/kube-co
+    </varlistentry>
+    <varlistentry>
+     <term>TCP</term>
+     <listitem>
+     <para>
+      If a TCP connection can be established is considered success.
+     </para>
+    </listitem>
+    </varlistentry>
+    <varlistentry>
+     <term>HTTP</term>
+     <listitem>
+     <para>
+      Any HTTP response between <literal>200</literal> and <literal>400</literal>
+      indicates success.
+     </para>
+    </listitem>
+    </varlistentry>
+   </variablelist>
+
+   <sect3 xml:id="sec.admin.monitoring.health.service.livenessprobe">
+    <title>livenessProbe</title>
+    <para>
+     <link xlink:href="https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/">livenessProbes</link>
+     are used to detect running but misbehaving pods/a service that
+     might be running (the process didn't die), but that is not responding as
+     expected.
+    </para>
+    <para>
+     Probes are executed by each <literal>kubelet</literal> against the pods that define them
+     and that are running in that specific node.
+    </para>
+    <para>
+     When a <literal>livenessProbe</literal> fails, &kube; will automatically restart the
+     pod and increase the <literal>RESTARTS</literal> count for that pod.
+    </para>
+    <para>
+     These probes will be executed every <literal>periodSeconds</literal> starting from
+     <literal>initialDelaySeconds</literal>.
+    </para>
+
+
+
+   </sect3>
+   <sect3 xml:id="sec.admin.monitoring.health.service.readinessprobe">
+    <title>readinessProbe</title>
+    <para>
+     <link xlink:href="https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/#define-readiness-probes">readinessProbes</link>
+     are used to wait for processes that take some time to start. Despite
+     the container running, it might be performing some time consuming
+     initializatoin operations. During this time, you don't want &kube; to route
+     traffic to that specific pod; also, you don't want that container to be
+     restarted because it will appear unresponsive.
+    </para>
+    <para>
+     These probes will be executed every <literal>periodSeconds</literal>
+     starting from <literal>initialDelaySeconds</literal> until the service is ready.
+    </para>
+    <para>
+     They support the same kind of probes as the <literal>livenessProbe</literal>
+    </para>
+
+    <para>
+     Both probe types can be used at the same time. The
+     <literal>livenessProbe</literal> will ensure that if a service is running
+     yet misbehaving, it will be restarted, and <literal>readinessProbe</literal>
+     will ensure that &kube; won't route traffic to that specific pod until it's
+     considered to be fully functional and running.
+    </para>
+   </sect3>
+  </sect2>
+
+  <sect2 xml:id="sec.admin.monitoring.health.general">
+   <title>General Health Checks</title>
+   <para>
+    It is advisable to apply other best practices from system administration
```

Comment:
Something we discussed once within the team without putting it into the style guide: @cjschroder recommended to use "We [as in SUSE] recommend to ..."

Reply:
Yes, I prefer the active to passive voice, "We recommend applying other best practices..."  It's not a big deal, but maybe we should discuss it one of these for the style guide.